tests/e2e/kserve_test.go

```diff
@@ -206,3 +210,63 @@ func (c *KserveTestCtx) validateDefaultCertsAvailable(t *testing.T) {
 	g.Expect(ctrlPlaneSecret.Type).Should(Equal(defaultIngressSecret.Type))
 	g.Expect(defaultIngressSecret.Data).Should(Equal(ctrlPlaneSecret.Data))
 }
+
+func (c *KserveTestCtx) ValidateKserveOwnsFeatureTrackers(t *testing.T) {
+	g := c.NewWithT(t)
+
+	fts := []string{
+		// c.ApplicationNamespace + "-kserve-external-authz",
+		c.ApplicationNamespace + "-serverless-serving-gateways",
+		c.ApplicationNamespace + "-serverless-serving-deployment",
+		c.ApplicationNamespace + "-serverless-net-istio-secret-filtering",
+	}
+
+	for _, ft := range fts {
+		g.Get(
+			gvk.FeatureTracker, types.NamespacedName{Name: ft},
+		).Eventually(300).Should(
```

Comment:
Giving the `.Eventually` a timeout value here gives it a chance to fail before the global timeout is hit, which causes a panic.

Reply:
mh, it should already have a timeout of 2 minutes